components/admin_console/workspace-optimization/dashboard.data.tsx

```diff
@@ -95,6 +98,7 @@ const useMetricsData = () => {
             id: 'admin.reporting.workspace_optimization.updates.description',
             defaultMessage: 'An update is available.',
         }),
+        hide: false,
```

Comment:
IMO we should omit this and make the property optional, so we don't have to set `false` manually

Reply:
ok